ert3/evaluator/_evaluator.py

```diff
@@ -122,9 +122,8 @@ def _fetch_results(ee_config, ensemble, stages_config):
 
 def _run(ensemble_evaluator):
     monitor = ensemble_evaluator.run()
-    for event in monitor.track():
-        if event.data is not None and event.data.get("status") in ["Stopped", "Failed"]:
-            monitor.signal_done()
```

Comment:
If you remove `signal_done` the evaluator never stops. It needs (or needed) the `USER_DONE` event. This is by design: the evaluation can complete, but that doesn't mean the user has received all events. So `signal_done` synchronizes the evaluator and the consumer.

Reply:
Yes, I know. Please, don't review this code yet, it is not working, and it is not going to stay like this.

Reply:
Okay, there's a draft status, maybe use that? I didn't catch the WIP in the title because I normally skip straight to the code.

On the other hand, I think discussion and review of code in all the stages (idea, prototype, draft, etc) is important.

Reply:
> On the other hand, I think discussion and review of code in all the stages (idea, prototype, draft, etc) is important.

Fully agree, and normally very welcome, but this pull request was just premature, so I closed it again...